### PR TITLE
Allows User to Globally Unsubscribe 

### PR DESCRIPTION
**Pull request by Vmack**
### What's this PR do?

This pull request provides GraphQL mutations so that a user will be able to unsubscribe to all email subscriptions 

### How should this be reviewed?

Is my resolver function correctly written? Am I using the proper arguments?

### Relevant tickets

References [Pivotal #171817299](https://www.pivotaltracker.com/story/show/171817299).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: src/repositories/northstar.js [hunk 1]
```diff
@@ -140,6 +140,39 @@ export const updateEmailSubscriptionTopics = async (
...
+  options,
+) => {
+  logger.debug('Updating email subscription status for user in Northstar', {
+    id,
+    emailSubscriptionStatus,
+  });
+  try {
+    const response = await updateUser(
+      id, {
+        email_Subscription_Status: emailSubscriptionStatus.subscribed
```
Comment (katiecrane) [inline src/repositories/northstar.js:162]: I think we just want to pass `emailSubscriptionStatus` here so like:
```suggestion
        email_Subscription_Status: emailSubscriptionStatus
```

Comment (Vmack) [reply src/repositories/northstar.js]: still doesn't work 😕but no worries, we'll figure this out.

File: src/repositories/northstar.js [hunk 2]
```diff
@@ -140,6 +140,38 @@ export const updateEmailSubscriptionTopics = async (
   return null;
 };
 
+/**
+ * 
+ * @param {String} id 
+ * @param {[EmailSubscriptionStatus]} emailSubscriptionStatus
+ * @param {Boolean} subscribed
```
Comment (mendelB) [inline src/repositories/northstar.js:147]: I think this can just be a single line:
```suggestion
 * @param {Boolean} emailSubscriptionStatus
```

**Comment by katiecrane** [comment]
@Vmack looks like you're also having some linting woes from Prettier! you can try running `npm run format` and then push up the changes from that and then hopefully your build will succeed!

**Pull request merged**